index.js

```diff
@@ -1,91 +0,0 @@
-const { randomInt } = require("crypto");
```

Comment:
It looks like there is an unused dependency here. It appears you could remove this line completely currently